tutorials/autotvm/tune_relay_x86.py

```diff
@@ -39,9 +39,12 @@
 # First we need to define the network in relay frontend API.
 # We can load some pre-defined network from :code:`relay.testing`.
 # We can also load models from MXNet, ONNX and TensorFlow.
+# Alternatively, we can `build a Graph Convolutional Network (GCN) with Relay
+# <https://docs.tvm.ai/tutorials/frontend/build_gcn.html#building-a-graph-convolutional-network>`
```

Comment:
Please use internal restructured text anchors for refering to something inside the doc